lib/web.xml

```diff
@@ -72,56 +72,12 @@
       <load-on-startup>3</load-on-startup>
     </servlet>
 
-    <servlet>
-      <servlet-name>HarvesterRegistrationLogin</servlet-name>
-      <servlet-class>edu.ucsb.nceas.metacat.harvesterClient.HarvesterRegistrationLogin</servlet-class>
-      <init-param>
-        <param-name>debug</param-name>
-        <param-value>1</param-value>
-      </init-param>
-      <init-param>
-        <param-name>listings</param-name>
-        <param-value>true</param-value>
-      </init-param>
-      <load-on-startup>3</load-on-startup>
-    </servlet>
-
-    <servlet>
-      <servlet-name>HarvesterRegistration</servlet-name>
-      <servlet-class>edu.ucsb.nceas.metacat.harvesterClient.HarvesterRegistration</servlet-class>
-      <init-param>
-        <param-name>debug</param-name>
-        <param-value>1</param-value>
-      </init-param>
-      <init-param>
-        <param-name>listings</param-name>
-        <param-value>true</param-value>
-      </init-param>
-      <load-on-startup>3</load-on-startup>
-    </servlet>
-
     <servlet>
         <servlet-name>AdvancedSearchServlet</servlet-name>
         <servlet-class>edu.ucsb.nceas.metacat.advancedsearch.AdvancedSearchServlet</servlet-class>
         <load-on-startup>3</load-on-startup>
     </servlet>
 
-    <!--
-    <servlet>
-      <servlet-name>HarvesterServlet</servlet-name>
-      <servlet-class>edu.ucsb.nceas.metacat.harvesterClient.HarvesterServlet</servlet-class>
-      <init-param>
-        <param-name>debug</param-name>
-        <param-value>1</param-value>
-      </init-param>
-      <init-param>
-        <param-name>listings</param-name>
-        <param-value>true</param-value>
-      </init-param>
-      <load-on-startup>3</load-on-startup>
-    </servlet>
-    -->
-    
     <!--
```

Comment:
Please add a very brief note saying why this is here and commented out (e.g. "This is an example for users who want to enable the OAI-PMH service - see: " ((add a link to docs))